src/runtime/runtime_impl.rs

```diff
@@ -104,13 +104,17 @@
 //! [idle]: struct.Runtime.html#method.shutdown_on_idle
 //! [`tokio::spawn`]: ../executor/fn.spawn.html
 
-use reactor::{Reactor, Handle, Background};
+use runtime::builder::Builder;
+use runtime::inner::Inner;
```

Comment:
I would suggest keeping `Inner` in this file (which will be `src/runtime/mod.rs`).

Reply:
Done